### ISSUE TITLE
Forum forsidens oversigt

### ISSUE THREAD

**Issue by Dustynt** [Nordic-T/nt-vanilla#7]
Synes ærligtalt forsiden er en rodebutik i forhold til det nuværende site. Der er alt for mange informationer, og de blå ikoner skaber efter min mening mere forvirring, end de skaber overblik.
Det samme gælder kategori og new ikonerne, den måde de ligger på skaber simpelthen for meget forvirring. Derudover synes "started by", "dato" og "clock" helt skal droppes, da det vil give mere overblik hvis det slet ikke fremgår.

Når jeg nu kommer med kritik, skal man selvfølgelig også komme med et løsningsforslag, og her er mit forslag på hvordan jeg synes det burde se ud i stedet, Forslaget er lavet i Photoshop, og kommer både med og uden blå ikoner så i kan se forskellen. 

Hvad siger i andre til det :)?

![nt-forumoversigt_gray_icons](https://cloud.githubusercontent.com/assets/13934115/9430141/5ef83a4e-49ea-11e5-9d5f-ff28873e2a11.jpg)

![nt-forumoversigt](https://cloud.githubusercontent.com/assets/13934115/9430143/69093592-49ea-11e5-87b6-6db2deb5a1a7.jpg)

**Comment by fbjerggaard** [comment]
Lækker feedback! Lad os endelig få noget feedback fra de andre testere på dette punkt

**Comment by Dustynt** [comment]
Mange tak. 
Det er så først nu det går på for mig, at jeg ikke vil ordnet views og comments, så de også står under hinanden :P Var faktisk meningen at de også skulle have stået på samme måde som alt andet, men i forhold til hvordan det er nu. Finder jeg personligt det her forslag mere overskueligt, da jeg går meget op i brugervenlighed ^^

**Comment by exetico** [comment]
Lad mig sige en ting: Annoncements forbliver blå. Husk at mange af de ting du ser i oversigten er tråde der er oprettet i eso, som ikke nødvendigvis skal være synlig i samme omfang på forsiden. 

Så hvad det angår, er der behovet for den blå farve til at fremhæve særlige ting. Resten kigger jeg lige på, men det var en kommentar til det første punkt :-)! Lige hurtigt fra mobilen. Kan forresten anbefale ForkHub til Android. Det virker OK.

**Comment by fbjerggaard** [comment]
Yes det er korrekt - lige pt. vises der alt for mange announcements på forsiden - de skal selvfølgelig rykkes ind så de kun bliver vist ved kategorien :)

**Comment by Dustynt** [comment]
@exetico 
Det har jeg fuld forståelse for. Men hvad så med at vælge en anden farve, som passer mere ind, det kan man sagtens samtidig med at den falder ud :)

Har lavet et eksempel med en anden farve som ses herunder. 
Jeg har også fået flyttet på comments og views, det kan sagtens justeres, og billede skal også kun ses som et eksempel på hvordan det vil se ud :)
![nt2](https://cloud.githubusercontent.com/assets/13934115/9430292/9a0ff748-49ef-11e5-8c11-e2fd80a6452e.jpg)

**Comment by exetico** [comment]
Lige en kort kommentar: Jeg er næsten sikker på at vi erstatter teksten til comments etc. med ikoner. 

GN - Jeg skal nok tilføje en kommentar i morgen :-)

**Comment by thei0173** [comment]
jeg er lidt in tvivl om ikke også tags ud for hver enkelt bruger bliver for meget.. ?
"member"
"sysops" etc.

**Comment by fbjerggaard** [comment]
@thei0173 - Men de er jo ikke på forsiden? :)

**Comment by Hamderjyden** [comment]
Jeg må give ret i at jeg syntes der generelt er ALT for mange farver. Hvis der skal stå at man er member syntes jeg blot det kan stå i en parantes eller lignende. Admin, moderator osv. kunne så have deres tags. 
For mange farver eksempel:
http://imgur.com/7dXaC2r

**Comment by exetico** [comment]
Members får også klart en passiv farve. Igen: Alle tilretninger er ikke lavet, men vi er rigtige glad for alle de ting der er kommet allerede nu!

**Comment by Dustynt** [comment]
I beder jo også om vores hjælp, så det får i ;)
Alt min kritik skal tags konstruktiv, og med henblik på brugervenlighed :)

**Comment by fbjerggaard** [comment]
@Dustynt Jeg har ryddet lidt op på forsiden samt rykket rundt på nogle elementer. Samtidig har jeg fjernet mange af de announcement topics fra forsiden så det giver et lidt mere realistisk billede. Kom gerne med noget feedback på det nye :)

**Comment by Dustynt** [comment]
Det ser en del bedre ud, men der kan stadigvæk godt gøres mere ved det, hvis du spørg mig :)

Personligt synes jeg <b>"started by"</b> fylder alt for meget, og er unødvendig", for du ser det alligevel når du går ind på den pågældende topic. Tænker derfor den lige så godt kan fjernes, den gør også at brugernavnet på den nyeste kommentar, nogle gange ryger ned og ligger under kategorien på det topic der er tale om.

Når ja <b>"Most recent"</b>, kan godt være det bare er mig, men synes det er en ret tung udtalelse.
Kunne man ikke i stedet gøre som på det nuværende site, og ændre det til: 

<b>{username} posted x {time} ago</b>

Hvis det kan lade sig gøre. Vil jeg gerne se hvordan det ser ud, da jeg godt nok har andre ting som kan gøres bedre. Men det kan være det løser sig selv, ved at ændre de to ting :)

**Comment by fbjerggaard** [comment]
Hmm.. Måske erstatte **started by** med **OP:** ? Eller er det bare fjollet?
Jeg tænker at erstatte **Most recent by** med **Latest poster: {username}**

**x time ago** bliver besværlig at implementere da vi så skal til at regne på datoer :P 

Hvilken opløsning sidder du på? Jeg skal helt ned på en 1024x768 før der er noget der ryger en linje ned, og så er det meget begrænset:

![image](https://cloud.githubusercontent.com/assets/1044024/9453564/7791645a-4abb-11e5-8fb5-78ebb37c58cb.png)

**Comment by Dustynt** [comment]
Jeg synes slet ikke der skal stå noget, eller også skal du fjerne dato og klokkeslet. Da jeg som tidligere nævnt et andet sted, mener at det bør fremgå på selve topic, hvor man kan se dato og klokken på OP og kommentarer.

<b>Lastest post: {username}</b> er en god ide :)

Jeg kører i 1900x1200, men har zoomet ind til 180% ellers er siden ALT for lille til mig.

**Comment by exetico** [comment]
Har du yderligere at tilføje pt? Ellers så lukker vi nemlig denne @Dustynt

**Comment by Dustynt** [comment]
Ja det har jeg :)
Jeg vil stadigvæk gerne se "New" lagt over i højre side ved siden af de to andre ikoner.
Jeg synes stadigvæk at New ikonet i venstre side, skaber mere forvirring og uoverskuelighed, end det gavner. Derfor vil jeg gerne se det på test over i højre side, for at se hvordan det ser ud, før i sender siden online til alle. Ved at lægge ikonet derover, vil i også få et mere sammenhængt layout. Derfor vil jeg sætte pris på at se ikonerne på test derover, så vi kan vurdere om det rent faktisk er bedre eller ej. :)

**Comment by exetico** [comment]
@fbjerggaard  - Hvordan ville du bedst fjerne "new" teksten, så der kun står count? Så kunne jeg godt smide den over til højre i stedet. Kan godt følge @Dustynt  her.

![skaermbillede 2015-10-25 kl 21 42 55](https://cloud.githubusercontent.com/assets/3549445/10717735/6c7af1e0-7b61-11e5-8859-dbf313ee062e.png)
